tox.ini

```
@@ -243,6 +243,12 @@ envlist =
     pypy3-test-instrumentation-pymysql
     lint-instrumentation-pymysql
 
+    ; opentelemetry-instrumentation-pymssql
+    py3{8,9,10,11,12,13}-test-instrumentation-pymysql
```

```suggestion
    py3{8,9,10,11,12,13}-test-instrumentation-pymssql
```
There are no tests in CI for pymssql

---

Good catch! I have updated.

---

@guillaumep thanks. Forgot to say you will need to run `tox -e generate-workflows` again and commit the changes

---

Done. There were changes in the resulting generated files that were unraleted to my work, I only added changes related to pymssql.

---

You should commit all changes made by tox -e generate-workflows

---

ok, PR updated will all changes.